sdk/docs/storage/README.md

```diff
@@ -81,18 +81,18 @@ Azure SDK for Embedded C is licensed under the [MIT][azure_sdk_for_c_license] li
 <!-- LINKS -->
 [azure_pattern_circuit_breaker]: https://docs.microsoft.com/azure/architecture/patterns/circuit-breaker
 [azure_pattern_retry]: https://docs.microsoft.com/azure/architecture/patterns/retry
-[azure_sdk_for_c_contributing]: ../../../CONTRIBUTING.md
+[azure_sdk_for_c_contributing]: https://github.com/Azure/azure-sdk-for-c/blob/master/CONTRIBUTING.md
 [azure_sdk_for_c_contributing_developer_guide]: ../../../CONTRIBUTING.md#developer-guide
```

Comment:
These also look to be unused.

Reply:
Right, that's why we did not fetch them out. Thanks! I will fix it for now.